vignettes/output_options.Rmd

```diff
@@ -0,0 +1,348 @@
+---
+title: "Output options"
+subtitle: "Get different kinds of output from OpenCage"
+author: "Daniel Possenriede, Jesse Sadler, Maëlle Salmon"
+date: "2020-10-07"
+description: >
+  "`oc_forward()`/`oc_reverse()` return lists of various type, namely data frames, JSON, GeoJSON or URLs, depending on the `return` value you specify. The possible `return` values are `df_list`, `json_list`, `geojson_list` and `url_only`."
+output: 
+  rmarkdown::html_vignette:
+    df_print: kable
+vignette: >
+  %\VignetteIndexEntry{Output options}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+
+
+{opencage} contains two types of main functions. 
+
+1. The `oc_forward_df()`/`oc_reverse_df()` functions, which always return a single tibble.
+2. The `oc_forward()`/`oc_reverse()` functions, which return lists of various types, depending on the `return` value you specify. The possible `return` values are `df_list`, `json_list`, `geojson_list` and `url_only`.
+
+We have shown the `oc_forward_df()`/`oc_reverse_df()` functions in the [Introduction to opencage](intro_opencage.html) and the [Customise your query](customise_query.html) vignettes. 
+The function arguments mentioned in the other vignettes are also available with `oc_forward()`/`oc_reverse()`, in general.
```

Comment:
why "in general"?

Reply:
`bind_cols` and `output` are not available in `oc_forward()`, because they do not apply there. (We only mention `output`, if I am not mistaken.)